src/cts/README.md

```diff
@@ -1,143 +1,138 @@
+no need to generate characterization data. On-the-fly characterization feature
+could still be optionally controlled by parameters specified to 
+`configure_cts_characterization` command. Use `set_wire_rc` command to 
 set clock routing layer.
 
 ## Commands
 
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
+
 ### Configure CTS Characterization
 
-```
-configure_cts_characterization [-max_slew <max_slew>] \
-                               [-max_cap <max_cap>] \
-                               [-slew_steps <slew_steps>] \
-                               [-cap_steps <cap_steps>]
+```tcl
+configure_cts_characterization 
+    [-max_slew max_slew]
+    [-max_cap max_cap]
+    [-slew_steps slew_steps]
+    [-cap_steps cap_steps]
 ```
 
-Argument description:
-
--   `-max_slew` is the max slew value (in seconds) that the characterization
-    will test. If this parameter is omitted, the code would use max slew value
-    for specified buffer in `buf_list` from liberty file.
--   `-max_cap` is the max capacitance value (in farad) that the
-    characterization will test. If this parameter is omitted, the code would
-    use max cap value for specified buffer in `buf_list` from liberty file.
--   `-slew_steps` is the number of steps that max_slew will be divided into
-    for characterization. If this parameter is omitted, the default is
-    12.
--   `-cap_steps` is the number of steps that max_cap will be divided into
-    for characterization. If this parameter is omitted, the default is 34.
+#### Options
 
+| Switch Name | Description | 
+| ----- | ----- |
+| `-max_slew` | Max slew value (in seconds) that the characterization will test. If this parameter is omitted, the code would use max slew value for specified buffer in `buf_list` from liberty file. |
+| `-max_cap` | Max capacitance value (in farad) that the characterization will test. If this parameter is omitted, the code would use max cap value for specified buffer in `buf_list` from liberty file. |
+| `-slew_steps` | Number of steps that `max_slew` will be divided into for characterization. (default 12) |
+| `-cap_steps` | Number of steps that `max_cap` will be divided into for characterization. (default 34) |
 
 ### Clock Tree Synthesis
 
-```
-clock_tree_synthesis -buf_list <list_of_buffers> \
-                     [-root_buf <root_buf>] \
-                     [-wire_unit <wire_unit>] \
-                     [-clk_nets <list_of_clk_nets>] \
-                     [-distance_between_buffers] \
-                     [-branching_point_buffers_distance] \
-                     [-clustering_exponent] \
-                     [-clustering_unbalance_ratio] \
-                     [-sink_clustering_enable] \
-                     [-sink_clustering_size <cluster_size>] \
-                     [-sink_clustering_max_diameter <max_diameter>]
+```tcl
+clock_tree_synthesis 
```

Comment:
Missing:
```
[-num_static_layers] \
[-sink_clustering_buffer] \
```

Reply:
I need some clarification on this, as I am not sure on the purpose on these switches.

Can you help me ping the tool maintainer?

Reply:
He is on vacation this week; we can keep a generic description and improve it later. This is good enough for now, thanks.